src/style/index.css

```diff
@@ -0,0 +1,20 @@
+@import "./reset.css";
+@import "./stars.css";
+@import "./card.css";
+
+:root {
+  --color-principal: #060b35;
+  --color-secondary: #616070;
+  --buy-button-principal-color: #00acdc;
+}
```

Comment:
you can also create a file variables.css and move all the variables to this file